clang/lib/Frontend/CompilerInvocation.cpp

```diff
@@ -4001,6 +4008,14 @@ bool CompilerInvocation::ParseLangArgs(LangOptions &Opts, ArgList &Args,
                         (T.isNVPTX() || T.isAMDGCN()) &&
                         Args.hasArg(options::OPT_fopenmp_cuda_mode);
 
+  // OpenACC Configuration.
+  if (Args.hasArg(options::OPT_fopenacc)) {
+    Opts.OpenACC = true;
```

Comment:
Maybe keep the version of the ACC in this value instead of having a separate string field, just like OpenMP does? 0 means th support is disabled, non-zero - supported version number.

Reply:
I likely will in the future, however as we currently don't have support for multiple versions of OpenACC, we only need 'true' or 'false'.  So for now, it seems prudent to not do that yet (as it won't be used).